### DISCUSSION TITLE
Should users be warned about enabling all the addons?

### DISCUSSION THREAD

**Discussion by jeffalo** [ScratchAddons/ScratchAddons#337] [Discussion]
a lot of users enable all the addons,

should this be warned against? i would say enabling all of them is unnecessary and could cause some slowdowns. 

we also wouldn't be able to easily tell which addons are enabled causing reported issues.

**Comment by jeffalo** [comment]
also, perhaps a list of enabled addons should be sent with feedback. i dont know how this would be possible without adding scratchaddons.com to the site manifest thing

**Comment by WorldLanguages** [comment]
I'd prefer adding warnings plus a "beta" tag.

> also, perhaps a list of enabled addons should be sent with feedback.

It could be added to the URL.

**Comment by GrahamSH-LLK** [comment]
I'm using all addons, and everything seems fine for me, although I haven't really been editing projects lately.
